src/types/types.ts

```diff
@@ -213,6 +210,11 @@ export type Phone = {
   nationalFormat?: string
 }
 
+export type HighlightStudyCardWithBlueBorder = {
```

Comment:
types.ts are for shared types that affect business logic. The types used for UI can/should live in the relevant files.

Reply:
Alright sounds good. I needed to use the same type in more than one file so I thought that it might be best to put it here.